src/coreclr/tools/Common/TypeSystem/Interop/IL/Marshaller.cs

```diff
@@ -1581,6 +1581,10 @@ protected override void EmitCleanupManaged(ILCodeStream codeStream)
 
     class AnsiStringMarshaller : Marshaller
     {
+        const int MAX_LOCAL_BUFFER_LENGTH = 260 + 1; // MAX_PATH + 1 
+
+        private ILLocalVariable _localBuffer = default;
```

Comment:
I think unused locals would break the build.

```suggestion
#if READYTORUN
        const int MAX_LOCAL_BUFFER_LENGTH = 260 + 1; // MAX_PATH + 1 

        private ILLocalVariable _localBuffer;
#endif
```